Update screenshots and steps for new UI architecture (#1234)
astronomer/docs: astro/deployment-metrics.md

> Located in the Workspace view of the Cloud UI, the **Analytics** page contains a suite of metrics for your Deployments. This page includes metrics that give you insight into the performance of both your data pipelines and infrastructure. Because metrics are collected in real time, you can use this page to detect irregularities in your pipelines or infrastructure as they happen.

Reviewer: Don't think this is the case? There is no **Analytics** page in the Workspace view anymore, unless you're talking about the main list of Deployments view?

    - Your Deployment was recently created and the Airflow webserver and scheduler are still spinning up.
    - Your Deployment's webserver and/or scheduler are restarting or otherwise not in a healthy, running state.

> If your Deployment is unhealthy, we recommend checking the status of your tasks and waiting for a few minutes. If your Deployment is unhealthy for more than 5 minutes, we recommend [reviewing scheduler logs](view-logs.md#view-airflow-scheduler-logs) in the Cloud UI or reaching out to [Astronomer support](https://cloud.astronomer.io/support).

Reviewer 2: ```suggestion
If your Deployment is unhealthy, Astronomer recommends checking the status of your tasks and waiting for a few minutes. If your Deployment is unhealthy for more than 5 minutes, [review the scheduler logs](view-logs.md#view-airflow-scheduler-logs) in the Cloud UI or contact [Astronomer support](https://cloud.astronomer.io/support).
```
